apps/frontend/src/components/autocompleteInput.tsx

```
@@ -0,0 +1,140 @@
+
+          {results.map((result, index) => (
+            <button
+              key={result.id}
+              className={cn(
+                'w-full px-3 py-2 text-left text-sm hover:bg-primary-foreground transition-colors',
+                'focus:bg-primary-foreground focus:outline-none',
+                selectedIndex === index && 'bg-primary-foreground'
+              )}
+              onClick={() => handleSelect(result)}
+              onMouseEnter={() => setSelectedIndex(index)}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)